app/models/git_repository.rb

```diff
@@ -8,6 +8,7 @@ class GitRepository < ApplicationRecord
 
   has_many :git_branches, :dependent => :destroy
   has_many :git_tags, :dependent => :destroy
+  before_destroy :delete_repo_dir
```

Comment:
@Fryguy That is not intentional. The delete is going to be delivered via the queue to the appliance that has the "git_owner" role enabled. If multiple users are doing an import/delete we could run into conflicts. Should all the git_repository methods be synchronized? since we could be getting requests from different appliances to the git_owner appliance?